### ISSUE TITLE
[ux/accessibility] Cmd + Click to open new tabs for links on the homepage

### ISSUE THREAD

**Issue by ktmud** [apache/superset#15912]
[Cmd + Click] should open links in new tabs. Now it doesn't work for most links on the home page.

**Comment by rusackas** [comment]
Since this is more feature request than bug report, and the thread has been silent for multiple years, we'll go ahead and close this out. If people are still encountering this in current versions (currently 3.x) please open a new "Ideas" discussion for the feature request or open a PR to address the problem.
